src/content/structured/components/breadcrumbs/code.mdx

```diff
@@ -19,35 +19,288 @@ tabs:
   ]
 ---
 
-import { IcBreadcrumbGroup, IcBreadcrumb } from "@ukic/react";
+import { IcBreadcrumbGroup, IcBreadcrumb, IcLink } from "@ukic/react";
+
+import { NavLink, MemoryRouter } from "react-router-dom";
 
 ## Component demo
 
-export const snippetsDefault = [
+export const snippets = [
   {
     language: "Web component",
     snippet: `<ic-breadcrumb-group>
-    <ic-breadcrumb page-title="Home" href="#"></ic-breadcrumb>
-    <ic-breadcrumb page-title="Beverages" href="# "></ic-breadcrumb>
-    <ic-breadcrumb current="true" page-title="Coffee" href="#"></ic-breadcrumb>
+  <ic-breadcrumb page-title="Home" href="#" />
+  <ic-breadcrumb page-title="Beverages" href="#" />
+  <ic-breadcrumb current="true" page-title="Coffee" href="#" />
 </ic-breadcrumb-group>`,
   },
   {
     language: "React",
     snippet: `<IcBreadcrumbGroup>
-    <IcBreadcrumb pageTitle="Home" href="#"></IcBreadcrumb>
-    <IcBreadcrumb pageTitle="Beverages" href="# "></IcBreadcrumb>
-    <IcBreadcrumb current="true" pageTitle="Coffee" href="#"></IcBreadcrumb>
+  <IcBreadcrumb pageTitle="Home" href="#" />
+  <IcBreadcrumb pageTitle="Beverages" href="# " />
+  <IcBreadcrumb current pageTitle="Coffee" href="#" />
 </IcBreadcrumbGroup>`,
   },
 ];
 
-<ComponentPreview snippets={snippetsDefault}>
+<ComponentPreview snippets={snippets}>
   <IcBreadcrumbGroup>
     <IcBreadcrumb pageTitle="Home" href="#" />
     <IcBreadcrumb pageTitle="Beverages" href="# " />
-    <IcBreadcrumb current="true" pageTitle="Coffee" href="#" />
+    <IcBreadcrumb current pageTitle="Coffee" href="#" />
   </IcBreadcrumbGroup>
 </ComponentPreview>
 
 <ComponentDetails component="ic-breadcrumb-group" />
```

Comment:
Please can we also add docs for ic-breadcrumb on this page (like we do with nav items on side nav etc), otherwise they are not covered anywhere